corehq/apps/users/models.py

```diff
@@ -2907,13 +2923,13 @@ def add_heartbeat(cls, domain, user_id, app_id, build_id, sync_date, device_id,
                     build_id,
                     sync_date, device_id,
                     app_version, num_unsent_forms, num_quarantined_forms,
-                    commcare_version, build_profile_id, last_heartbeat
+                    commcare_version, build_profile_id, last_heartbeat, fcm_token
                 ) VALUES (
                     %(domain)s, %(user_id)s, %(app_id)s, CLOCK_TIMESTAMP(), CLOCK_TIMESTAMP(),
                     %(build_id)s,
                     %(sync_date)s, %(device_id)s,
                     %(app_version)s, %(num_unsent_forms)s, %(num_quarantined_forms)s,
-                    %(commcare_version)s, %(build_profile_id)s, CLOCK_TIMESTAMP()
+                    %(commcare_version)s, %(build_profile_id)s, CLOCK_TIMESTAMP(), %(fcm_token)s
```

Comment:
Do we need to worry about where these records are parsed? For older records, this column would simply be null in the table?

Reply:
> Do we need to worry about where these records are parsed? 

I am afraid I do not understand by `where these records are parsed`.

> For older records, this column would simply be null in the table?

Yes

Reply:
Got it . I don't think we need to worry as older records will have this value as null